packages/go_router/example/lib/query_params.dart

```diff
@@ -41,55 +42,16 @@ class App extends StatelessWidget {
             HomeScreen(families: Families.data),
         routes: <GoRoute>[
           GoRoute(
-            path: 'family/:fid',
-            builder: (BuildContext context, GoRouterState state) =>
-                FamilyScreen(
-              family: Families.family(state.params['fid']!),
-            ),
-            routes: <GoRoute>[
-              GoRoute(
-                path: 'person/:pid',
-                builder: (BuildContext context, GoRouterState state) {
-                  final Family family = Families.family(state.params['fid']!);
-                  final Person person = family.person(state.params['pid']!);
-                  return PersonScreen(family: family, person: person);
-                },
-              ),
-            ],
-          ),
+              path: 'family/:id',
+              builder: (BuildContext context, GoRouterState state) {
+                return FamilyScreen(
+                  family: Families.family(state.params['id']!),
+                  privacyMode: state.queryParams['privacy'] == 'true',
```

Comment:
For this sample, it might be better to choose a more common scenario such as a filter `?filter=abc` or search query.